### DISCUSSION TITLE
what to do with stone

### DISCUSSION THREAD

**Discussion by lstebner** [jeremyckahn/farmhand#203] [Ideas]
mining will leave players with lots of stone, what can we do with it? some ideas were [discussed in discord](https://discord.com/channels/714539345050075176/847884323330850847/883885352908107836) of varying difficulties/scales. just going to list them out here for some discussion

- constructing a house
- constructing an "improved" cowpen where cows generate hearts faster (requires largest cowpen)
- constructing a pond for raising aquatic creatures (fish for fishing, or maybe snails, or something else)
- creating "statues" which add buffs to the player i.e. statue of harvest for increase yields for plots. statues can offer small-ish buffs that stack

the idea behind all of these is that they would require a rather large amount of stone, since stone is fairly common when mining. this also requires the player to have bought several inventory upgrades to be able to store it all.

**Comment by jeremyckahn** [comment]
I particularly like the idea of crafting statues that provide a variety of buffs to the player, I think there's a lot we could do with that!
